### ISSUE TITLE
Controller works but has massive input lag

### ISSUE THREAD

**Issue by Alhanalem** [MTCKC/ProconXInput#19]
I was able to get everything set up and working but there is huge input lag- at least a full second between taking any actions and them registering. Can I do something about this?

**Comment by ImFalling** [comment]
Did you remember to set the analog stick's center by pressing the "share" button, like the CMD prompt told you to?

**Comment by Alhanalem** [comment]
Yes, i followed all the instrucitons, the software said "Now entering fast
input loop" after pressing the button, it was still extremely laggy

**Comment by theAeon** [comment]
I have the exact same issue.

**Comment by theAeon** [comment]
Update: plugging into a different usb port fixed.

**Comment by Alhanalem** [comment]
Perhaps it must be a USB 3.0 port?

Stragently the ports on the front of my system should be USB 3.0 (and they
are blue) but they seem to be acting as 2.0.
But... a game controller really shouldnt need more than USB 2.0's bandwidth.

**Comment by theAeon** [comment]
It's entirely possible it's a latency thing- I plugged it into my front panel USB 3.0 ports and it lagged, but plugging it into the motherboard's usb 3 ports works fine. If I had to guess, my front panel ports are not provided by the chipset, but the back ports are.

**Comment by coztellation** [comment]
I got it working on Windows 10 by going into "Details" on Task Manager, and right clicking ProconXInput.exe, setting the Priority to Realtime. This instantly fixed it for me.

**Comment by Alhanalem** [comment]
I tried this suggestion, but no luck :( I've also tried different USB ports
to no avail as well.

**Comment by davidsides** [comment]
Oddly, I don't get horrible delay using ProconXInput, but I tested the pro controller using my own program and I also suffered the horrible delay (in USB and Bluetooth), I tried every USB port to get it to work but port itself didn't have much effect.

What caused it for me was my Logitech Unifying Usb Receiver, the moment I unplugged it the delay vanished. Instead of just not using it, I uninstalled the device from the device manager (called Logitech Unifying Usb Receiver) which fixed the delay and for some reason still allows my keyboard to work. (Though it likes to come back on its own, so not really a great solution)

Maybe some other device connected to your computer is causing the delay for you.

**Comment by MoundN** [comment]
It's BULLSHIT. One day my usb c slot has 0 delay when playing with my procon and another day itt has like 5 seconds of delay... 

Why is that sso. where is  the consisttency. probably some ffucking windows issue.

**Comment by FrickTown** [comment]
@MoundN Have you tried using Steam Input instead? Works flawlessly with my procon.
